core/src/main/scala/org/apache/spark/Accumulators.scala

```diff
@@ -271,6 +307,65 @@ class Accumulator[T] private[spark] (
   }
 }
 
+case class UpdateInfo private[spark](val rddId: Int, splitId: Int)
```

Comment:
It will be good also if you add some documentation to this class, as you use it in mapWithAccumulator
tell me what it's usage